antora-playbook.yml

```diff
@@ -57,6 +57,7 @@ asciidoc:
     image-puller-operator-name: Kubernetes Image Puller Operator
     image-puller-operator-id: kubernetes-image-puller-operator
     image-puller-operator-repository-name: kubernetes-image-puller-operator
+    image-puller-deployment-id: kubernetes-image-puller-operator
```

Comment:
could you point me where this property is used?

Reply:
@ibuziuk sure!
https://github.com/eclipse/che-docs/edit/master/modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc#L74

>. The `eclipse-che namespace` contains these deployments: `{image-puller-deployment-name}` and `{image-puller-deployment-id}`.

Reply:
ack. makes sense